### PR TITLE
[12.0] [ADD] Add  shopinvader_cart_reminder

### PR DESCRIPTION
**Pull request by lmignon**
forward port of #330

### CODE REVIEW

File: shopinvader_pending_cart_reminder/data/ir_cron.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record forcecreate="True" id="ir_cron_shopinvader_sale_reminder" model="ir.cron">
+        <field name="name">Shopinvader sales - Reminder (not confirmed sales)</field>
```
Comment (rousseldenis) [inline shopinvader_pending_cart_reminder/data/ir_cron.xml]: /sShopinvader sales/Shopinvader cart instead?

File: shopinvader_pending_cart_reminder/data/mail_template.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    <record id="mail_template_shopinvader_sale_reminder" model="mail.template">
+        <field name="name">Shopinvader sale reminder</field>
+        <field name="email_from">${(object.shopinvader_backend_id.email or '')|safe}</field>
+        <field name="subject">Reminder ${object.name}</field>
+        <field name="partner_to">${object.partner_id.id}</field>
+        <field name="model_id" ref="sale.model_sale_order"/>
+        <field name="auto_delete" eval="False"/>
+        <field name="lang">${object.partner_id.lang}</field>
+        <field name="body_html"><![CDATA[ To customize ]]></field>
```
Comment (rousseldenis) [inline shopinvader_pending_cart_reminder/data/mail_template.xml]: Why not writing a default one ?

**Review by rousseldenis — COMMENTED** [review]
LGTM. Code review. Some questions

**Comment by lmignon** [comment]
@rousseldenis What do you think about the last changes?

**Comment by lmignon** [comment]
@sebastienbeau @Cedric-Pigeon @acsonefho What do you think about the default email template?

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by acsonefho — APPROVED** [review]
LGTM.
Email template seems ok too.

**Pull request merged**